### ISSUE TITLE
Inspector resizes window when left-clicking inspected list item

### ISSUE THREAD

**Issue by pamoroso** [Interlisp/medley#1059]
**Describe the bug**
In the inspector, when inspecting `'(A B C D)` and clicking the `D` entry in the left column, the window goes into resize mode and a window frame that follows the mouse is overlaid, as if selecting `Shape` from the right-click menu. The issue is reproducible.

**To Reproduce**
1. sign into your Interlisp Online account
2. under `Initial Exec`, select `Interlisp`
3. click `Run Medley`
4. in the Executive Window, evaluate `'(INSPECT '(A B C D))`
5. select `Inspect` and position the window
6. in the left column, click `4`

**Expected behavior**
`4` is highlighted in reverse and no window resize frame appears.

**Screenshots**
The inspector window with the resize frame after clicking `4`:
![InspectorResize1](https://user-images.githubusercontent.com/10342479/215485629-637c4d1f-fc0c-4a18-b9ea-0b8616f2a248.png)

The full desktop:
![InspectorResize2](https://user-images.githubusercontent.com/10342479/215485632-d1f2ca39-6def-4274-8862-96ac581770d0.png)

**Context (please complete the following information):**
 - OS: chromeOS Stable
 - OS Version: 109.0.5414.94
 - Host arch: x86_64 (ASUS Chromebox 3)
 - Maiko version: not sure how to get it
 - IL:MAKESYSDATE: 18-Dec-2022 20:59:08

**Additional context**
The issue occurs also when first left-clicking `2` and then `3`.

**Comment by rmkaplan** [comment]
Yes, I think this is an issue with detecting the nearest corner in MODERNIZE.  The problem is that the general inspector (also the array inspector) creates a window that it then completely fills up with its data, and so it is very difficult to distinguish whether the mouse has gone down in the few points of the border region or on the edge of the last element.  I suppose a brute force fix would be to add a few extra border points at the bottom of these windows so that there is a little more discriminating space for the mouse.  Maybe this should be done for all windows, but I have been reluctant to mess generically with the deeper innards of windows.  Maybe careful alignments of other windows will be screwed up.

The border regions of Tedit window are also a little obscure—line pointing is a narrow region on the left of the window, splitting is a narrow region on the right. I think I heuristically defined it so that those regions start up a few points from the actual bottom, giving some room for corner discrimination.

The exec windows have different problems.  First, I have never been able to figure out why, but sometimes a new exec window is created without modernization having been applied, even though I think I caught all of those interfaces.  Second, TTYIN magically (for me) manages the type-in portion of the exec window.  If the type-in line is completely empty, between events, then the modernize mouse function for that window is invoked.  But once a single character has been typed, TTYIN takes control and the window’s mouse function is ignored.  This could be fixed if we could figure out where/how TTYIN grabs the mouse—I haven’t been able to do that.

**Comment by nbriggs** [comment]
Isnt there a window border global we could set a little bigger?

**Comment by rmkaplan** [comment]
Note also that bumping that parameter (WBorder) also increases the width of the black part of the border, not just the internal white margin, by the same amount.  Possibly, WBorder could be extended to be a pair (black white) so that these could be controlled independently.

**Comment by masinter** [comment]
Changing the cursor to reflect the behavior of the mouse click would likely not have layout consequences.

**Comment by rmkaplan** [comment]
That should also be done, so the user would know what is about to happen.  But it is still a problem, I think, that the target area is so small.

**Comment by masinter** [comment]
surely there is some subset of 7 possible shift meta control + left click that isn't already taken? 
Then we could assign that chord+left click to mean shape/move always. If you're not close to a corner or edge it's move; if you're close to a corner it means shape, otherwise you're close to an edge, it means stretch.

This would eliminate the ambiguity
